devshell.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs }:
+
+with pkgs;
```

Comment:
I recommend you fold this into the flake.nix file, unless, you also want to support devshell without nix's flake.
It's just one more file people have to worry about

Reply:
Yeah good point. Let me give this a shot with copilot. I'm sure it's straightforward. There's not really anything special I'm using from devshell.